src/main/java/uk/gov/hmcts/ccd/domain/service/createcase/SubmitCaseTransaction.java

```diff
@@ -106,6 +106,9 @@ private CaseDetails saveAuditEventForCaseDetails(Event event, CaseType caseType,
         auditEvent.setSecurityClassification(securityClassificationService.getClassificationForEvent(caseType,
                                                                                                      eventTrigger));
         auditEvent.setDataClassification(savedCaseDetails.getDataClassification());
+        if(eventTrigger.getSignificantItem()!=null) {
```

Comment:
Formatting: Missing spaces on either sides of `!=`

Reply:
Addressed